### PR TITLE
Resolves issue #15120

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- Resolves issue #15120.

### CODE REVIEW

**Comment by sgiehl** [comment]
Thanks for the PR. We also need to update the composer.lock in this case.

**Comment by peter279k** [comment]
Hi @sgiehl, thanks for your reply.

My question is: How to update the `composer.lock` file?

Using the `php-5.5.x`  to execute `composer` and using update command to update cached dependency information on `composer.lock` file?

**Comment by sgiehl** [comment]
actually you would need to let composer update the composer.json & composer.lock by executing those commands (without your changes). The PHP version you are running doesn't matter as long as it satisfies the requirements (>5.5.x):
```
composer remove facebook/xhprof
composer require --dev lot/xhprof dev-master
```
After that I guess the additional repository entry for facebook/xhprof can be removed manually from composer.json

**Comment by peter279k** [comment]
Hi @sgiehl, I've update the `composer.lock` file and please review this PR again.

Thanks.

File: composer.json [hunk 1]
```diff
@@ -54,10 +54,10 @@
     "require-dev": {
         "aws/aws-sdk-php": "2.7.1",
         "phpunit/phpunit": "~4.8",
-        "facebook/xhprof": "dev-master",
         "phpseclib/phpseclib": "~0.3.8",
         "symfony/var-dumper": "~2.6.0",
-        "symfony/yaml": "~2.6.0"
+        "symfony/yaml": "~2.6.0",
+        "lox/xhprof": "dev-master"
```
Comment (sgiehl) [inline composer.json]: could you also remove the package definition for `facebook/xhprof` the lines below (63-88).

Comment (peter279k) [reply composer.json]: I've removed them.

**Review by sgiehl — APPROVED** [review]

**Comment by sgiehl** [comment]
@peter279k  I'll close this one, as we won't change that for 3.x-dev branch anymore. Would you mind recreating the PR against 4.x-dev branch, if it still makes sense? Thx!

**Comment by peter279k** [comment]
@sgiehl, thanks for your reply. I will recreating one PR for `4.x-dev` branch :).

**Pull request closed without merging**